src/settings/index.js

```diff
@@ -2,7 +2,8 @@ import validate from './validate'
 import humps from 'lodash-humps'
 import { get, defaultsDeep } from 'lodash'
 
-const json = require(`./${process.env.REACT_APP_SETTINGS_FILE}`)
+const settingsFile = get(process.env, 'REACT_APP_SETTINGS_FILE', 'production')
```

Comment:
we can actually just use the `NODE_ENV` variable 

```suggestion
const json = require(`./${process.env.NODE_ENV}`)
```